ksm.go

```diff
@@ -100,10 +100,10 @@ const (
 )
 
 var ksmSettings = map[ksmMode]ksmSetting{
-	ksmOff:        {false, 1000, 500}, // Turn KSM off
-	ksmSlow:       {true, 500, 100},   // Every 100ms, we scan 1 page for every 500 pages available in the system
-	ksmStandard:   {true, 100, 10},    // Every 10ms, we scan 1 page for every 100 pages available in the system
-	ksmAggressive: {true, 10, 1},      // Every ms, we scan 1 page for every 10 pages available in the system
+	ksmOff:        {1000, 500, false}, // Turn KSM off
```

Comment:
next I'm expecting some fixup here for filling out 'un-named fields' or somesuch...

Reply:
Quite. ftr, I'm seeing slightly odd behaviour with `gometalinter` - sometimes it will run clean but the next run will show issues. But fwics the linters themselves have not changed.

Reply:
@jodh-intel Maybe we should `go get` a specific version of gometalinter instead of master in our CI.